templates/org/create.tmpl

```diff
@@ -15,28 +15,27 @@
 						<span class="help">{{.i18n.Tr "org.org_name_helper"}}</span>
 					</div>
 
-					<div class="inline required field {{if .Err_OrgVisibility}}error{{end}}">
```

Comment:
Why remove required?

Reply:
It puts stars after the labels for radioboxes too.

Anyway, it's a radio-box which is initially checked so there's no easy way to uncheck it - so of course it's required.

Reply:
If it's really required I can mangle it back in.

Reply:
Done